api/src/main/resources/workbench-api.yaml

```diff
@@ -498,6 +498,19 @@ paths:
             application/json:
               schema:
                 type: boolean
+  /v1/account/trainings-enabled:
+    get:
+      tags:
+        - profile
+      summary: Whether trainings are enabled for this user. For use during the training migration.
```

Comment:
it would be nice to mention this is temp, so that its know this will be removed

Reply:
Good idea, done.